app/src/main/java/org/fossasia/openevent/general/MainActivity.kt

```diff
@@ -74,4 +77,12 @@ class MainActivity : AppCompatActivity() {
             else -> super.onBackPressed()
         }
     }
+
+    private fun showSnackbar() {
+        val textSnackbar = intent.getStringExtra("snackbar_message")
```

Comment:
Extract to a constant string "snackbar_message"

Reply:
I have to change textSnackbar to snackbar_message ?